### ISSUE TITLE
Work links in summaries should not be included in the download

### ISSUE THREAD

**Issue by nianeyna** [nianeyna/ao3downloader#50]
Regression caused by #43. Previously, the downloader only recognized internal/relative work links (of the form `/works/12345`). Now, work links of the form `[...]/works/12345` are included also. This has the unintended side effect of including *absolute* work links (`https://archiveofourown.org/works/12345`) in the list. This can happen when an author manually links to another work on ao3 in their summary. Aside from not being intended behavior, the ao3 root url is still prepended to the absolute links (`https://archiveofourown.orghttps://archiveofourown.org/works/12345`) causing mayhem.

tl;dr amend the work and series patterns to recognize internal (starting with "/") links only.

**Comment by nianeyna** [comment]
Expressions are in `strings.py` and are named `AO3_WORK` and `AO3_SERIES`

Work pattern should match:

`/works/12345`
`/collections/CollectionName/works/12345`

Should not match:

`https://archiveofourown.org/works/12345`
`works/12345`
`/fooworks/12345`

Series expression should follow the same rules but with "series" instead of "works"
